app/Jobs/NotifyUserOfNewGistComment.php

```diff
@@ -53,7 +53,8 @@ private function sendNotificationEmail($comment, $gist, $user)
                 'comment' => $comment,
                 'gist' => $gist,
                 'user' => $user,
-                'body' => $parser->parse($comment['body'])
+                'unsubscribeLink' => $this->getUnsubscribeUrl($user),
```

Comment:
Is there any reason we can't just call this off the user in the view?

Reply:
If the user isn't found, there's a "can't call that on null" exception thrown. We can check for the user's existence above this array instead of in a separate function, too.

Reply:
Yah, I'm just figuring this notification email should probably not be sent if there is no user, yah?

Reply:
Just remembered why I added that check: the tests pass `$user = null` to the job during setup. I removed the method and added an inline check.